src/controllers/recipe/recipe.statics.ts

```diff
@@ -50,3 +52,35 @@ export const findAllRecipesLikedByUser = async ({
   }).exec();
   return results;
 };
+
+export const deleteRecipeById = async ({
```

Comment:
One thing we forgot to do is add this function signature to the `IRecipeModel` interface. Copy and paste the function name and the signature from here to the IRecipeModel in `recipe.types.ts`, following the same pattern already there.